yarn.lock

```diff
@@ -1312,6 +1312,11 @@
     signedsource "^1.0.0"
     yargs "^9.0.0"
 
+"@guardian/pasteup@1.0.0-alpha.8":
```

Comment:
Any idea why this is introduced as part of this PR?

Reply:
No idea. I noticed it was introduced automatically as I ran make dev. Should I remove it from the PR?

Reply:
I was surprised to see this here as I would only expect to see a change to the `yarn.lock` when dependencies have updated in the `package.json`, and none have in this PR. However I believe this update to the `yarn.lock` is actually correct - It looks like the `pasteup` dependencies were updated in an earlier PR https://github.com/guardian/dotcom-rendering/pull/529 but the `yarn.lock` wasn't updated to correspond with these changes.

Reply:
I'll leave it in then 👍